src/components/Trade/TradeInput.tsx

```diff
@@ -604,6 +605,9 @@ export const TradeInput = () => {
 
   const handleInputAssetClick = useCallback(
     (action: AssetClickAction) => {
+      // prevent opening the asset selection while they are being populated
+      if (!isSwapperInitialized) return
```

Comment:
last part of the fix - prevent users clicking asset selection while swapper manager is being initialised. This prevents empty asset list during init and then full asset list on second open.